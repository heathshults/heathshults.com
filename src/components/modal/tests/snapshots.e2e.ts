import { snapIt } from '../../../test';

const component = 'modal';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<hs-modal>default</hs-modal>');
    snap('renders as open', '<hs-modal open>default</hs-modal>');
    snap('renders dismissible button and overlay', '<hs-modal open dismissible>default</hs-modal>');
  });
});
