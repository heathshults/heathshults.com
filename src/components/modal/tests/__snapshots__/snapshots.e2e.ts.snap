// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal renders renders as open 1`] = `"<hs-modal open=\\"\\" class=\\"hydrated\\"><div aria-hidden=\\"\\" class=\\"hs-overlay hs-overlay--fullpage hs-overlay--visible\\"></div><div role=\\"dialog\\" class=\\"hs-modal hs-modal--visible\\">default</div></hs-modal>"`;

exports[`modal renders renders correctly with no props 1`] = `"<hs-modal class=\\"hydrated\\"><div aria-hidden=\\"\\" class=\\"hs-overlay hs-overlay--fullpage\\"></div><div role=\\"dialog\\" class=\\"hs-modal\\">default</div></hs-modal>"`;

exports[`modal renders renders dismissible button and overlay 1`] = `"<hs-modal open=\\"\\" dismissible=\\"\\" class=\\"hydrated\\"><div aria-hidden=\\"\\" class=\\"hs-overlay hs-overlay--fullpage hs-overlay--visible\\"></div><div role=\\"dialog\\" class=\\"hs-modal hs-modal--visible\\"><button type=\\"button\\" class=\\"hs-button hs-button--close\\">Ã—</button>default</div></hs-modal>"`;
